plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -678,6 +683,28 @@ nothrow:
         }
     }
 
+    void opCall(ref IncreaseFilePrio data) {
+        try {
+            spinSql!(() @trusted {
+                string fileContent = readText(conf.metadataPath);
+                auto jContent = parseJSON(fileContent);
+                auto objectData = jContent.object["file-prio"];
+                Path[] prioFiles;
+                foreach (prioFileElem; objectData.array) {
+                    string strData = prioFileElem.toString;
+                    strData = strData.replace("\\", "");
+                    strData = strData.replace("\"", "");
+                    prioFiles ~= Path(strData);
+                }
+                foreach (prioFilePath; prioFiles) {
+                    db.mutantApi.increaseFilePrio(prioFilePath);
```

Comment:
only this one should be inside spinSql because the other lines may fail of various reasons such as the file not existing, wrong json format etc